### DISCUSSION TITLE
Hotspot Missing from Snippet

### DISCUSSION THREAD

**Discussion by architektpetr** [google/model-viewer#3454] [Q&A]
Hi. Code snippet doesn't update after placing hotpots in a model viewer (Chrome, Edge...). I have to inspect the viewer and get the snippet manually. Is it a correct way or am I missing something important? Thanks, Petr

**Comment by samaneh-kazemi** [comment]
Hi @architektpetr, thanks for reaching out. Can you please share more details? A glitch link with your code/usage would be helpful

**Comment by elalish** [comment]
Are you referring to the editor? Perhaps https://github.com/google/model-viewer/issues/3455?

**Comment by architektpetr** [comment]
Yes, I'm referring to the editor. The issue seems to be the same.

Dne pá 13. 5. 2022 21:28 uživatel Emmett Lalish ***@***.***>
napsal:
